StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkVerification/NetworkingLinkVerificationViewController.swift

```diff
@@ -168,63 +168,45 @@ extension NetworkingLinkVerificationViewController: NetworkingOTPViewDelegate {
-                            guard let self = self else { return }
-                            switch result {
-                            case .success(let networkedAccountsResponse):
-                                let networkedAccounts = networkedAccountsResponse.data
-                                if networkedAccounts.isEmpty {
-                                    self.dataSource.analyticsClient.log(
-                                        eventName: "networking.verification.success_no_accounts",
-                                        pane: .networkingLinkVerification
-                                    )
-                                    self.requestNextPane(manifest.nextPane)
-                                } else {
-                                    self.dataSource.analyticsClient.log(
-                                        eventName: "networking.verification.success",
-                                        pane: .networkingLinkVerification
-                                    )
-                                    self.requestNextPane(.linkAccountPicker)
-                                }
-                            case .failure(let error):
-                                self.dataSource
-                                    .analyticsClient
-                                    .logUnexpectedError(
-                                        error,
-                                        errorName: "FetchNetworkedAccountsError",
-                                        pane: .networkingLinkVerification
-                                    )
-                                self.dataSource
-                                    .analyticsClient
-                                    .log(
-                                        eventName: "networking.verification.error",
-                                        parameters: [
-                                            "error": "NetworkedAccountsRetrieveMethodError",
-                                        ],
-                                        pane: .networkingLinkVerification
-                                    )
-                                self.requestNextPane(manifest.nextPane)
-                            }
-
-                            // only hide loading view after animation
-                            // to next screen has completed
-                            DispatchQueue.main.asyncAfter(
-                                deadline: .now() + 1.0
-                            ) { [weak view] in
-                                view?.showLoadingView(false)
-                            }
-                        }
+                case .success:
+                    dataSource.analyticsClient.log(
+                        eventName: "networking.verification.success",
+                        pane: .networkingLinkVerification
+                    )
+                    requestNextPane(.linkAccountPicker)
                 case .failure(let error):
```

Comment:
this was a duplicated `case`? surprised the compiler doesn't complain about that

Reply:
I think this might be a case of a difficult-to-read diff? There was an extra API call removed that wasn't doing much (web code didn't have it anymore, so removed it). Particularly difficult to read the Diff if whitespaces was turned off for this particular case

Reply:
ah, that was it. I see it now, thanks